### PR TITLE
Logic to handle file downloading during test execution changed

### PR DESCRIPTION
**Pull request by helen-dikareva**
It's require for https://github.com/DevExpress/testcafe/pull/855
/cc @AlexanderMoskovkin

### CODE REVIEW

File: src/test-run/command.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ export default {
     setStepsSharedData:               'set-steps-shared-data',
     getStepsSharedData:               'get-steps-shared-data',
     getAndUncheckFileDownloadingFlag: 'get-and-uncheck-file-downloading-flag',
-    uncheckFileDownloadingFlag:       'uncheck-file-downloading-flag',
     nativeDialogsInfoSet:             'native-dialogs-info-set',
-    takeScreenshot:                   'take-screenshot'
+    takeScreenshot:                   'take-screenshot',
+    // NOTE: we need this message to support new page unload barier's mechanism
```
Comment (AlexanderMoskovkin) [inline src/test-run/command.js]: I think this comment is redundant. [this](https://github.com/DevExpress/testcafe-legacy-api/pull/18/files#diff-a21f56bee759db22062e32250e3df48cR161) is enough

**Review by AlexanderMoskovkin — CHANGES_REQUESTED** [review]

**Review by AlexanderMoskovkin — APPROVED** [review]
lgtm

**Review by AndreyBelym — APPROVED** [review]
lgtm

**Pull request merged**